src/components/EditableToDo.vue

```diff
@@ -15,18 +18,18 @@
           and you will not be able to undo this action!
         </p>
         <div class="flex flex-row space-x-6 mb-2 font-semibold">
-          <button
-            class="rounded-full bg-green px-2 text-white"
+          <ModalButton
```

Comment:
Please give to this component a more generic name like BaseButton and use it for every button not just in the popup